uiowa/uiowa
# [Charts] Added charts feature

# How to test 

Related to https://github.com/uiowa/uiowa/issues/5368. 
Resolves https://github.com/uiowa/uiowa/issues/9309. 

```
ddev composer install && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local config-split:activate charts -y && ddev drush @sandbox.local uli --uid=286  /node/add/page
```
Code review

1. Add a new page
2. Edit layout
3. Add a charts block
4. Upload CSV option below or find another CSV to upload and test.
5. Choose "column" or "bar" or "line" chart
6. Add `$` in value prefix field or `%` in suffix field.  
7. Save and view chart and verify that the chart, tooltip, legend display as expected. 
8. Run Siteimprove plugin on the page and confirm that there are no `AA` or `S` violations.


- [salaries_states.csv](https://github.com/user-attachments/files/23340732/salaries_states.csv)
- [salaries.csv](https://github.com/user-attachments/files/23340734/salaries.csv)
- [salary_engineering.csv](https://github.com/user-attachments/files/23508108/salary_engineering.csv)
